### ISSUE TITLE
"Example" is read out loud by the screen reader at the very end of the example

### ISSUE THREAD

**Issue by rautenburger** [swiss/styleguide#554]
Also visual impaired users should be informed when an example begins and when it ends. 
The text "Example" is however read out when the example already passed. Furthermore, the visual impaired user should also informed when the example is finished as it is not apparent without visual input. 

Tested on:
https://q-src.github.io/styleguide/de/
Tested with:
NVDA screen reader

**Comment by bit-pro-iew-eui** [comment]
Please explain where exactly in the guideline

**Comment by rautenburger** [comment]
Everywhere where there is an Example Box. For instance here:
![image](https://user-images.githubusercontent.com/31723894/31371922-c81c64dc-ad93-11e7-8c2f-7f0a27925549.png)

The text "example" is read out at the very last position. It should be at the first position, so that the screen reader knows that an/the example is following.
